config/locales/mailers/offline_cancellation_mailer.en.yml

```diff
@@ -0,0 +1,4 @@
+en:
+  offline_cancellation_mailer:
+    send_notification:
+      subject: Your reservation for %{instrument} has been canceled
```

Comment:
I think that's fine for now. I'm honestly not that crazy about the GS structure. I think you end up with too many deeply nested and tiny files. I like what we've started to do here where we keep controller and view locales in the same file. But this is a special mailer, so I'm not sure where the best place for it is.